packages/website/components/link/link.js

```diff
@@ -4,7 +4,8 @@ import Link from 'next/link';
 
 const WrappedLink = ({ tabIndex = 0, href, ...otherProps }) => (
   <Link href={href} {...otherProps}>
-    <a href="replace" {...otherProps} tabIndex={tabIndex} onClick={otherProps.onClick}>
+    {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */}
+    <a {...otherProps} tabIndex={tabIndex} onClick={otherProps.onClick}>
```

Comment:
Will `next/link` render the correct `href` on the anchor tag if no href is provided?  If so, it looks good to me.  

I noticed web3 was using `href="replace"` all over the place prior to extracting the anchors into this file, so I wrongly assumed that the correct href was being rendered